src/app/signalhandler.cpp

```diff
@@ -93,15 +91,14 @@ namespace
         QMetaObject::invokeMethod(qApp, [] { QCoreApplication::exit(); }, Qt::QueuedConnection);  // unsafe, but exit anyway
     }
 
-#ifdef STACKTRACE
     void abnormalExitHandler(const int signum)
     {
         const char msg[] = "\n\n*************************************************************\n"
             "Please file a bug report at https://bug.qbittorrent.org and provide the following information:\n\n"
             "qBittorrent version: " QBT_VERSION "\n\n"
             "Caught signal: ";
         const char *sigName = sysSigName[signum];
-        const std::string stacktrace = getStacktrace();
+        const std::string stacktrace = boost::stacktrace::to_string(boost::stacktrace::stacktrace());
```

Comment:
Off topic: since we're already creating a signal-unsafe `std::string` here, we could simplify this implementation by just building up one string instead of an array of `char *` that we then loop over with `std::for_each`. I would be happy to do this in another PR if there are no objections.

Reply:
IMO we should strive to cut down signal-unsafe operations, not the other way.